### PR TITLE
hw5.3 solution problem with maven i do not know how to fix

### PR DESCRIPTION
**Pull request by Andrewmazyar**

### CODE REVIEW

**Comment by Andrewmazyar** [comment]
i have problem with maven he give me Error ...... :21:27: Local variable name 'wList' must match pattern '^[a-z]([a-z0-9][a-zA-Z0-9]*)?$'. [LocalVariableName]
in method replaceAll i was also use \\W - doesn work i need help

**Comment by Tarabarnik** [comment]
> i have problem with maven he give me Error ...... :21:27: Local variable name 'wList' must match pattern '^[a-z](%5Ba-z0-9%5D%5Ba-zA-Z0-9%5D*)?$'. [LocalVariableName]
> in method replaceAll i was also use \W - doesn work i need help

you should have more than one small letter before a big one:
wList - false
filteredWords - true

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -9,6 +14,25 @@
...
+            for (String word: lineList) {
+                if (word.toLowerCase().startsWith("w")) {
+                    wordWList.append(word.replaceAll("\\W","")).append(" ");
+                }
+            }
+            if (wordWList.length() > 0) {
+                result = wordWList.toString().split(" ");
+            } else {
+                result = new String[0];
+            }
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:31]: try to solve without this `else` block

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -9,6 +14,21 @@
...
+        try {
+            String line = Files.readString(Paths.get(fileName));
+            String[] lineList = line.toLowerCase().split(" ");
+            StringBuilder wordWList = new StringBuilder();
+            for (String word: lineList) {
+                if (word.toLowerCase().startsWith("w")) {
+                    wordWList.append(word.replaceAll("\\W","")).append(" ");
+                }
+            }
+            result = wordWList.length() > 0 ? wordWList.toString().split(" ") : new String[0];
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:27]:
```suggestion
            result = wordWList.toString().split(" ");
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -9,6 +14,21 @@
...
+            for (String word: lineList) {
+                if (word.toLowerCase().startsWith("w")) {
+                    wordWList.append(word.replaceAll("\\W","")).append(" ");
+                }
+            }
+            result = wordWList.toString().split(" ");
+            Arrays.sort(result);
+            return wordWList.length() > 0 ? result : new String[0];
+        } catch (IOException e) {
+            throw new RuntimeException("No such file");
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:31]:
```suggestion
            throw new RuntimeException("No such file", e);
```

**Pull request closed without merging**